### PR TITLE
Update searching.md

### PR DESCRIPTION
**Pull request by RumovZ**

### CODE REVIEW

File: src/searching.md [hunk 1]
```diff
@@ -405,5 +409,5 @@ Note and card IDs can be found in the [card info](stats.md) dialog in the
 browser. These searches may also be helpful when doing add-on
 development or otherwise working closely with the database.
 
-Object IDs will not work in the mobile clients, and are not intended to
+Object IDs will not work in AnkiMobile, and are not intended to
```
Comment (RumovZ) [inline src/searching.md:412]: Is this still true for AnkiMobile? AnkiDroid supports it.

File: src/searching.md [hunk 2]
```diff
@@ -284,10 +292,6 @@ cards that have moved into relearning more than 3 times
 `prop:ease!=2.5`  
 cards easier or harder than default
 
-Note that due only matches review cards and learning cards with an
-interval of a day or more: cards in learning with small intervals like
-10 minutes are not included.
-
```
Comment (RumovZ) [inline src/searching.md:290]: This must have changed at some point, but I didn't investigate with which version.

File: src/searching.md (hunk 1 above)
Comment (dae) [reply src/searching.md:412]: They've been supported for some time, we could probably just drop this paragraph.

**Comment by dae** [comment]
Thanks Rumo!

**Pull request merged**